bundles/org.openhab.binding.meross/src/main/resources/OH-INF/thing/msg100.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="meross"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="msg100">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Meross MSG100</label>
+		<description>Meross Garage Door</description>
+
+		<channels>
+			<channel id="doorState" typeId="doorState"/>
```

Comment:
Please stick to the `lower-case-hyphen` naming convention for channel id's, and channel type id's

Reply:
I will change that.